Kaguya/Kaguya/Discord/Commands/Administration/Mute.cs

```diff
@@ -45,17 +46,20 @@ public class Mute : KaguyaBase<Mute>
         }
 
         [Command(RunMode = RunMode.Async)]
-        [Summary("Mutes a user with an optional reason.")]
+        [Summary("Mutes a user permanently with an optional reason.")]
```

Comment:
The user is muted indefinitely not permanently, unless you have no unmute feature